### PR TITLE
Remove unnecessary dependencies

### PR DESCRIPTION
**Pull request by belanagy**

### CODE REVIEW

File: sccameraview/src/main/java/com/example/sccameraview/Camera1View.java [hunk 1]
```diff
@@ -64,7 +63,7 @@ void openCamera() {
             camera.setPreviewTexture(getSurfaceTexture());
             camera.startPreview();
         } catch (IOException e) {
-            Timber.e(e);
+            Log.d("SCCameraView", e.getMessage());
```
Comment (WonderCsabo) [inline sccameraview/src/main/java/com/example/sccameraview/Camera1View.java]: Please move this to a constant somewhere.

Use the appropriate log level, as with Timber. For example here you should use `Log.e`.

File: sccameraview/src/main/java/com/example/sccameraview/Camera2View.java [hunk 2]
```diff
@@ -122,9 +123,11 @@ public void openCamera() {
             return;
         }
 
-        new RxPermissions(activity)
-                .request(VIDEO_PERMISSIONS)
-                .subscribe();
+        for (String permission : VIDEO_PERMISSIONS) {
```
Comment (WonderCsabo) [inline sccameraview/src/main/java/com/example/sccameraview/Camera2View.java]: Please remove the whole permission check. The client should do this.

File: sccameraview/src/main/java/com/example/sccameraview/Camera2View.java [hunk 3]
```diff
@@ -372,6 +378,8 @@ private static Size chooseVideoSize(Size[] choices) {
         return choices[choices.length - 1];
     }
 
+
```
Comment (WonderCsabo) [inline sccameraview/src/main/java/com/example/sccameraview/Camera2View.java]: Remove these empty lines.

**Review by WonderCsabo — CHANGES_REQUESTED** [review]

File: sccameraview/src/main/java/com/example/sccameraview/Camera1View.java [hunk 4]
```diff
@@ -64,7 +63,7 @@ void openCamera() {
             camera.setPreviewTexture(getSurfaceTexture());
             camera.startPreview();
         } catch (IOException e) {
-            Timber.e(e);
+            Log.e(getContext().getString(R.string.app_name), e.getMessage());
```
Comment (WonderCsabo) [inline sccameraview/src/main/java/com/example/sccameraview/Camera1View.java]: Use a Java final static field instead of a string resource.

**Review by WonderCsabo — CHANGES_REQUESTED** [review]
Please fix the string constants.

File: sccameraview/src/main/java/com/example/sccameraview/Camera1View.java (hunk 4 above)
Comment (belanagy) [reply sccameraview/src/main/java/com/example/sccameraview/Camera1View.java]: ok, done

**Review by WonderCsabo — APPROVED** [review]

**Pull request merged**